apiv2/db_import/importers/tomogram.py

```diff
@@ -9,9 +9,15 @@
 
 
 class TomogramItem(ItemDBImporter):
-    # TODO - add the alignment_id field once that data is added the first time.
-    # id_fields = ["run_id", "tomogram_voxel_spacing_id", "deposition_id", "alignment_id", "processing", "reconstruction_method""]
-    id_fields = ["run_id", "tomogram_voxel_spacing_id", "deposition_id", "processing", "reconstruction_method"]
+    id_fields = [
+        "alignment_id",
```

Comment:
I'm not 100% sure we can add `alignment_id` here yet, let me check on that!

Reply:
I can revert it to be commented for now..